### PR TITLE
add note about enabling and verifying Ansible role

### PR DESCRIPTION
**Pull request by cbudz**
Hey Suyog,

I've added a note about enabling the Ansible role and verifying status of the worker. While I was in the file I also took the opportunity to make some corrections, ie, lower-case for playbooks. 

You can find a preview of the changes here:

http://file.rdu.redhat.com/~cbudzilo/CloudForms/ansible-note/build/tmp/en-US/html-single/#automation_management_providers

Please let me know what you think. 

-Chris

### CODE REVIEW

File: doc-Managing_Providers/topics/Automation_Management_Providers.adoc [hunk 1]
```diff
@@ -1,30 +1,42 @@
 [[automation_management_providers]]
 = Automation Management Providers
 
+[NOTE]
+====
+{product-title} ships with the Ansible role disabled. Navigate to menu:settings menu[Configuration > Settings] and select the desired server under *Zones*. Set the *Server Role* for *Embedded Ansible* to *On* to enable the role.
```
Comment (suyogsainkar) [inline doc-Managing_Providers/topics/Automation_Management_Providers.adoc]: @cbudz Can you rephrase the first line to something like "In {product-title}, the Ansible role is disabled by default." What do you think? We use "include(s|d)" instead of "ship" as in the style guide, though that also can be avoided here I think.

Comment (suyogsainkar) [inline doc-Managing_Providers/topics/Automation_Management_Providers.adoc]: "Navigate to menu:settings menu[Configuration > Settings] and select the desired server under *Zones* > As the settings menu is not an actual UI element, we write it in the following form, and not include it in "menu:". Example: "Navigate to the settings menu, then Configuration → Access Control → Groups to administer groups."

File: doc-Managing_Providers/topics/Automation_Management_Providers.adoc [hunk 2]
```diff
@@ -1,30 +1,42 @@
...
+* Add a source control repository that contains your playbooks.
 * Establish credentials with your inventory. 
-* Back your services, alerts and policies using available Playbooks. 
+* Back your services, alerts and policies using available playbooks. 
+
+[[verifying-embedded-ansible-worker-state]]
+=== Verifying the Embedded Ansible Worker State
+Verify that the Embedded Ansible worker has started to utilize its features. 
+
+. Navigate to menu:settings menu[Configuration > Diagnostics] and click on the desired server.
```
Comment (suyogsainkar) [inline doc-Managing_Providers/topics/Automation_Management_Providers.adoc]: As the settings menu is not an actual UI element, we write it in the following form, and not include it in "menu:". Example: "Navigate to the settings menu, then Configuration → Access Control → Groups to administer groups."

File: doc-Managing_Providers/topics/Automation_Management_Providers.adoc [hunk 3]
```diff
@@ -1,30 +1,42 @@
...
 
-* Add a source control repository that contains your Playbooks.
+* Add a source control repository that contains your playbooks.
 * Establish credentials with your inventory. 
-* Back your services, alerts and policies using available Playbooks. 
+* Back your services, alerts and policies using available playbooks. 
+
+[[verifying-embedded-ansible-worker-state]]
+=== Verifying the Embedded Ansible Worker State
+Verify that the Embedded Ansible worker has started to utilize its features.
```
Comment (suyogsainkar) [inline doc-Managing_Providers/topics/Automation_Management_Providers.adoc]: I'm not sure if Embedded Ansible qualifies as a proper name and should be capitalized.

**Comment by cbudz** [comment]
Hey Suyog,

I updated the doc based on your suggestion. Thanks for bringing up the matter of the settings menu. I've made a note for future reference.

-Chris

**Comment by suyogsainkar** [comment]
@cbudz Awesome, thanks for making those changes, Chris. Looking good to me; merging to master.
